plugins/gs/src/components/deployments/DeploymentsDataProvider/utils.ts

```diff
@@ -43,6 +43,28 @@ export type DeploymentData = {
   app?: string;
 };
 
+function getStatus(deploymentStatus: string) {
+  const successfullStatuses = ['reconciled', 'deployed'];
```

Comment:
Since "successful" is written with one l only, I think this should be named `successfulStatuses`